app/javascript/controllers/globus_controller.js

```diff
@@ -0,0 +1,42 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+
+  static targets = ['requestGlobusLink', 'stagingLocation']
+
+  async createDestination() {
+    this.setRequesting()
+
+    const response = await fetch('/globus', {method: 'post'})
+
+    if (response.ok) {
+      const dest = await response.json()
+      this.setFinishedRequest(dest)
+      this.setDestination(dest)
+    } else {
+      console.log('Unable to create Globus Destination:', response)
+      throw new Error('Unable to create Globus Destination')
+    }
```

Comment:
does throwing the error show anything to the user ?

Reply:
It shows that it is an invalid staging location.

Reply:
Oh actually it doesn't -- sorry I was initially confused and thinking this was in the Rails controller. I can have it throw an alert, which won't be the prettiest but at least will get noticed.

Reply:
I misread your comment initially and thought this was about the Rails controller, not the Stimulus one! If it's ok, I'd like to address this in a follow on ticket #1340  to move this work along, since it is holding up other work.